src/server/services/hli/hli.ts

```diff
@@ -210,18 +216,69 @@ async function fetchRegelingen(authProfileAndToken: AuthProfileAndToken) {
   return aanvragenResponse;
 }
 
+async function fetchSpecificaties(
+  authProfileAndToken: AuthProfileAndToken
+): Promise<ApiResponse<HLIRegelingSpecificatieFrontend[]>> {
+  const response = await fetchZorgnedAanvragenHLI(
+    authProfileAndToken.profile.id
+  );
+  if (response.status !== 'OK') {
+    return response;
+  }
+
+  const aanvragen = response.content.reduce(
+    (
+      filteredAanvragen: ZorgnedAanvraagWithRelatedPersonsTransformed[],
+      aanvraag
+    ) => {
+      const documents = aanvraag.documenten.filter(
+        (d) => d.title === 'AV-RTM Specificatie'
```

Comment:
Kun je dit in een const zetten? `const RTM_SPECIFICATIE_TITLE = 'AV-RTM Specificatie'